phpBB/phpbb/console/command/db/list_command.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+namespace phpbb\console\command\db;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class list_command extends \phpbb\console\command\db\migration_command
+{
+	protected function configure()
+	{
+		$this
+			->setName('db:list')
+			->setDescription($this->user->lang('CLI_DESCRIPTION_DB_LIST'))
+			->addOption(
+				'uninstalled',
+				'u',
+				InputOption::VALUE_NONE,
+				$this->user->lang('CLI_MIGRATION_ONLY_UNINSTALLED')
+			)
+		;
+	}
+
+	protected function execute(InputInterface $input, OutputInterface $output)
+	{
+		$show_installed = !$input->getOption('uninstalled');
+		$installed = $uninstalled = array();
+
+		foreach ($this->load_migrations() as $name)
+		{
+			if ($this->migrator->migration_state($name) !== false)
+			{
+				$installed[] = $name;
+			}
+			else
+			{
+				$uninstalled[] = $name;
+			}
+		}
+
+		if (!empty($installed) && $show_installed)
+		{
+			$output->writeln('<info>' . $this->user->lang('CLI_MIGRATION_INSTALLED') . $this->user->lang('COLON') . '</info>');
+			$output->writeln($installed);
+		}
+
+		if (!empty($installed) && $show_installed && !empty($uninstalled))
+		{
+			$output->writeln('');
+		}
+
+		if (!empty($uninstalled))
+		{
+			$output->writeln('<info>' . $this->user->lang('CLI_MIGRATION_UNINSTALLED') . $this->user->lang('COLON') . '</info>');
```

Comment:
I don't think uninstall is a good name here: thes migrations can be both uninstalled and not installed. I'd prefer the more generic `not installed`or `available` here